myuw_vue/components/home/major-declaration/declare-major.vue

```diff
@@ -207,25 +198,35 @@ export default {
     isJunior() {
       return this.classLevel === 'JUNIOR';
     },
+    isSenior() {
+      return this.classLevel === 'SENIOR';
+    },
+    isSophomore() {
+      return this.classLevel === 'SOPHOMORE';
+    },
     notDeclaredMajor() {
       // MUWM-5261
       return this.noDeclaredMajor(this.termMajors);
     },
-    showCard() {
-      return (this.isJunior && this.notDeclaredMajor && this.seattle &&
-        (this.isNoticeFetching || this.isProfileFetching ||
-         this.showContent));
+    isTargetViewer() {
+      return (
+        this.seattle && (this.isJunior || this.isSenior) ||
```

Comment:
@fanglinfang I removed bothell from here so it does not yet show to bothell students.